src/Controller/Api/PersonController.php

```diff
@@ -494,6 +499,26 @@ public function getPerson(Person $person): Response
             'country' => $person->getCountry(),
             'organization' => $person->getOrganization(),
             'position' => $person->getPosition()
-        ]);
+        );
+
+        $account = $person->getAccount();
+        if ($account instanceof Account) {
+            $posix = $account->isPosix();
+            $posixUsername = $account->getUsername();
+        }
+
+        $isMe = (($currentAccount instanceof Account) && $person->getId() === $currentAccount->getPerson()->getId()) ? true : false;
+        $authenticatedData = array(
+            'isposix' => $posix,
+            'posixUsername' => $posixUsername,
```

Comment:
$posixUsername should be defined as null or empty string, It only exists if $account is an instanceOf Account

Reply:
Are you proposing we always send a null for the 3 protected fields instead of just not including them in the API response?

Reply:
good catch, thx